core/src/main/scala/zio/jdbc/package.scala

```diff
@@ -39,7 +39,7 @@ package object jdbc {
   def delete(sql: SqlFragment): ZIO[ZConnection, Throwable, Long] =
     for {
       connection <- ZIO.service[ZConnection]
-      result     <- connection.executeSqlWith(sql)(_.executeLargeUpdate())
+      result <- ZIO.scoped(connection.executeSqlWith(sql)(_.executeLargeUpdate()))
```

Comment:
This one I buy because we want to close the statement after executing and retrieving the number of updated rows. 👍